examples/contract-transfer/lib.rs

```diff
@@ -65,7 +65,9 @@ pub mod give_me {
     #[cfg(test)]
     mod tests {
         use super::*;
+        use ink_env::test::transfer_in;
```

Comment:
I don't like that we have to import this and not even use it (since it's used from within the macro). Can we instead import this in the macro?

Reply:
Makes sense, yeah